packages/language-server/test/plugins/typescript/features/CompletionProvider.test.ts

```diff
@@ -191,6 +219,7 @@ describe('CompletionProviderImpl', function () {
 
         assert.deepStrictEqual(eventCompletions, <CompletionItem[]>[
             {
+                commitCharacters: [],
```

Comment:
Empty array and undefined should behave the same but won't be overridden by `itemDefaults`